### ISSUE TITLE
File size increase from prebid 0.18 to prebid 0.21.

### ISSUE THREAD

**Issue by nanek** [prebid/Prebid.js#1125]
## Type of issue

Large file size increase from prebid 0.18 to prebid 0.21. Looking for ways to minimize the file size of prebid.js

## Description

I've isolated the cause to #962 which increased the size of prebid minified built without adapters from 40k to 55k. 

## Steps to reproduce

```
gulp build --adapters adapters-none.json
# view file size
# remove imports in polyfill and try again.
```

## Platform details
prebid 0.21

**Comment by mkendall07** [comment]
Yeah I thought that was the case. I'm not sure why those poly fills need to be so large. They might be bringing the kitchen sink with them as well, it should be looked at. Feel free to propose a solution here, other than going back to the manual poly fills I'm not sure what else can be done.
